code/go/0chain.net/conductor/services/allocations.go

```diff
@@ -0,0 +1,80 @@
+package services
+
+import (
+	"0chain.net/conductor/stores"
+	"0chain.net/conductor/types"
+	"0chain.net/conductor/utils"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var allocationStore = stores.GetAllocationStore()
+
+type AllocationService struct {
+	allocation *types.Allocation
+	baseUrl    string
+}
+
+func NewAllocationService(baseUrl string) *AllocationService {
+	return &AllocationService{
+		allocation: &types.Allocation{},
+		baseUrl:    baseUrl,
+	}
+}
+
+func (s *AllocationService) CompareAllocationsValue() (bool, error) {
+	// Read allocationID from file
```

Comment:
Please move this to a separate private (unexported) function within AllocationService and use it here.